js/index.js

```diff
@@ -0,0 +1,19 @@
+//FOOTER
+let today = new Date();
+let thisYear= today.getFullYear();
+let footer = document.querySelector('footer');
+let copyright = document.createElement('p');
+copyright.innerText = `Nicole Bradley ${thisYear}`;
+footer.appendChild(copyright);
+
+//SKILLS SECTION
+let skills = ["WordPress", "Elementor", "JavaScript", "HTML", "CSS", "Nodejs", "MongoDB"];
+let skillsSection = document.getElementById("skills");
+//I know I was supposed to query the UL element via the skillsList and not the whole document but I couldnt get it to work 
```

Comment:
Can you provide an example of what you tried that didn't work? The best way is to make a branch and reference that, so everyone can be sure they're looking at the same code, but for short things, a code snippet in a comment may be enough.